src/caselawclient/Client.py

```diff
@@ -753,6 +771,19 @@ def get_properties_for_search_results(self, judgment_uris: list[str]) -> str:
         response = self._send_to_eval(vars, "get_properties_for_search_results.xqy")
         return decode_multipart(response)
 
+    def search_and_parse_results(
```

Comment:
What we currently call `perform_advanced_search` in public and editor uis:

https://github.com/nationalarchives/ds-caselaw-public-ui/blob/6c966bba962e3a8efdd951029b68fa9cb576b3e5/judgments/utils.py#L42

and 

https://github.com/nationalarchives/ds-caselaw-editor-ui/blob/fd8f1e9100a8a60f081eecb79836f44688e70c94/judgments/utils/perform_advanced_search.py